src/main/java/de/hhu/abschlussprojektverleihplattform/repository/LendingRepository.java

```diff
@@ -138,7 +140,7 @@ public LendingEntity getLendingByProductAndUser(ProductEntity product, UserEntit
     @Override
     public List<LendingEntity> getAllLendingsForUser(UserEntity user) {
         String query = "SELECT * FROM LENDING_ENTITY WHERE BORROWER_USER_ID="
-            + user.getUserId();
+                + user.getUserId();
```

Comment:
wieso hast du das weiter eingerückt?
pass bitte deine settings in intellij an, damit alle indentations nur 4 zeichen lang sind